internal/pkg/cli/init.go

```diff
@@ -41,33 +41,41 @@ type initVars struct {
 	// Flags unique to "init" that's not provided by other sub-commands.
 	shouldDeploy   bool
 	appName        string
-	svcType        string
+	wkldType       string
 	svcName        string
 	dockerfilePath string
+	image          string
 	imageTag       string
 	port           uint16
+
+	schedule string
```

Comment:
nit: can we also add comments in the struct for fields that are specific for jobs vs. services